doc/release/v2_3_70_2.md

```diff
@@ -5,6 +5,16 @@ YARP 2.3.70.2 (UNRELEASED) Release Notes                            {#v2_3_70_2}
 A (partial) list of bug fixed and issues resolved in this release can be found
 [here](https://github.com/robotology/yarp/issues?q=label%3A%22Fixed+in%3A+YARP+v2.3.70.2%22).
 
+Important Changes
+-----------------
+
+### Libraries
+
+#### `YARP_OS`
+
+* Unification of code in `DGramTwoWayStream` for the allocation of write
```

Comment:
I like if there is an issue regarding the change that we put the original issue number in brackets, like 
`[...]and read buffers (#899).

Reply:
I would go for the complete url `https://github.com/robotology/yarp/issues/889` rather then just the issue number `#889` . It is easier to render in all kind of markdown visualizers, and on GitHub is rendered in the same way ( https://github.com/robotology/yarp/issues/889 vs #889 ) . If in the future we will migrate to a different issue tracker, the regular structure of the issue URL will make for an easy target for renaming all the links.